packages/drawer/src/Drawer.tsx

```diff
@@ -144,15 +149,19 @@ export default function Drawer({
           role="dialog"
           showCloseButton={showCloseButton}
         >
-          <DrawerContent tabIndex={-1}>{children}</DrawerContent>
+          {children}
```

Comment:
Can you explain these changes? Is this a refactor, as the title of the PR suggests, or does this fixes something?

Reply:
This PR is mainly about removing the paddings and the internal container from the `Drawer` component in order to make it more generic. Why? Essnetially to remove the hacks we had to apply in the new drawers

Reply:
Since we ended up closing the PR that changed the hacks, should we close this PR as well? We can always tackle this later.